packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -873,10 +874,14 @@ detail: ${JSON.stringify(eventDetail)}`);
     content: string
   ): AcsChatMessage {
     const senderId = graphMessage.from?.user?.id || undefined;
+    let messageType = 'chat';
+    if (graphMessage?.policyViolation) {
+      messageType = 'blocked';
```

Comment:
this should set the link property too so that the user can read the DLP docs:
```suggestion
      messageType = 'blocked';
      link: 'https://go.microsoft.com/fwlink/?LinkId=2132837';
```

Ref: https://azure.github.io/communication-ui-library/iframe.html?viewMode=docs&id=ui-components-messagethread--message-thread#blocked-message